buildroot-external/board/jethome/rootfs-overlay/usr/lib/systemd/system/jethome-swap.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=HassOS File swap for JetHome
```

Comment:
This is mostly used for the 512Mb version of the controller. But in some cases on 1Gb RAM you can't do without swap on the disk.
My idea: check existance of /mnt/data/swapfile file and if exists enable (or create) swap. I don't have better idea too. This can be fixed later when a better way exists. 
Maybe move file to another folder?

Reply:
> you can't do without swap on the disk.

Even with zram? What happens exactly? I haven't tested 512MB machines recently, but we definitely had machines which booted and worked without swap.

> This can be fixed later when a better way exists.

I doubt that anyone will work on it once its merged. So it seems, this machine is under powered to deliver the "Home Assistant OS" experience as is. Adding swap might help fixing it, but it should be implemented in a generic and maintainable way.

Reply:
Ok. I removed all code for swap https://github.com/home-assistant/operating-system/pull/1621/commits/2a93b002807dc2370af0e1bbbf1786baaf2d35a4